### PR TITLE
[API] [Documentation] Payments endpoint

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | yes |
| Related tickets | |
| License         | MIT |

### CODE REVIEW

File: docs/api/payments.rst [hunk 1]
```diff
@@ -1,102 +1,242 @@
...
+| _links[self]           | Link to itself                                                              |
++------------------------+-----------------------------------------------------------------------------+
+| _links[payment-method] | Link to related payment method                                              |
++------------------------+-----------------------------------------------------------------------------+
+| _links[order]          | Link to related order                                                       |
++------------------------+-----------------------------------------------------------------------------+
+
+.. note::
+
+    Read more about :doc:`Payment in the component docs</components/Payment/models>`.
```
Comment (TheMadeleine) [inline docs/api/payments.rst]: Payments

File: docs/api/payments.rst [hunk 2]
```diff
@@ -1,102 +1,242 @@
...
+------------------------------
+
+If you request a payment via API, you will receive an object with the following fields:
+
++------------------------+-----------------------------------------------------------------------------+
+| Field                  | Description                                                                 |
++========================+=============================================================================+
+| id                     | Unique id of the payment                                                    |
++------------------------+-----------------------------------------------------------------------------+
+| method                 | :doc:`The payment method object serialized </api/payment_methods>` for cart |
```
Comment (TheMadeleine) [inline docs/api/payments.rst]: for the cart

File: docs/api/payments.rst [hunk 3]
```diff
@@ -1,102 +1,242 @@
...
++------------------------+-----------------------------------------------------------------------------+
+| amount                 | The amount of payment                                                       |
++------------------------+-----------------------------------------------------------------------------+
+| state                  | :doc:`State of the payment process </book/orders/payments>`                 |
++------------------------+-----------------------------------------------------------------------------+
+| _links[self]           | Link to itself                                                              |
++------------------------+-----------------------------------------------------------------------------+
+| _links[payment-method] | Link to related payment method                                              |
```
Comment (TheMadeleine) [inline docs/api/payments.rst]: Link to **the** related payment method

File: docs/api/payments.rst [hunk 4]
```diff
@@ -1,102 +1,242 @@
...
++------------------------+-----------------------------------------------------------------------------+
+| _links[order]          | Link to related order                                                       |
```
Comment (TheMadeleine) [inline docs/api/payments.rst]: Link to **the** related order

File: docs/api/payments.rst [hunk 5]
```diff
@@ -1,102 +1,242 @@
...
+
+    The *id = 20* is just an example. Your value can be different.
+
+Exemplary Response
+^^^^^^^^^^^^^^^^^^
 
 .. code-block:: text
 
-    STATUS: 200 OK
+     STATUS: 200 OK
```
Comment (TheMadeleine) [inline docs/api/payments.rst]: is this additional space needed here?

File: docs/api/payments.rst [hunk 6]
```diff
@@ -1,102 +1,242 @@
...
++--------------------+----------------+--------------------------------------------------------------------+
+| Parameter          | Parameter type | Description                                                        |
++====================+================+====================================================================+
+| Authorization      | header         | Token received during authentication                               |
++--------------------+----------------+--------------------------------------------------------------------+
+| page               | query          | *(optional)* Number of the page, by default = 1                    |
++--------------------+----------------+--------------------------------------------------------------------+
+| limit              | query          | *(optional)* Number of items to display per page, by default = 10  |
++--------------------+----------------+--------------------------------------------------------------------+
+| sorting[amount]    | query          | *(optional)* Order of sorting on amount field (desc/asc)           |
```
Comment (TheMadeleine) [inline docs/api/payments.rst]: ```Sorting direction on the ``amount`` field (``DESC``/``ASC``)```

File: docs/api/payments.rst [hunk 7]
```diff
@@ -1,102 +1,242 @@
...
++--------------------+----------------+--------------------------------------------------------------------+
+| sorting[createdAt] | query          | *(optional)* Order of sorting on created at field (asc by default) |
```
Comment (TheMadeleine) [inline docs/api/payments.rst]: ``` Sorting direction on the ``createdAt`` field (``ASC`` by default) ```

File: docs/api/payments.rst [hunk 8]
```diff
@@ -1,102 +1,242 @@
...
++--------------------+----------------+--------------------------------------------------------------------+
+| sorting[updatedAt] | query          | *(optional)* Order of sorting on updated at field (desc/asc)       |
```
Comment (TheMadeleine) [inline docs/api/payments.rst]: ``` Sorting direction on the ``updatedAt`` field (``DESC``/``ASC``) ```

File: docs/api/payments.rst [hunk 9]
```diff
@@ -1,102 +1,242 @@
...
++--------------------+----------------+--------------------------------------------------------------------+
 
-    STATUS: 200 OK
+Example
+^^^^^^^
 
-.. code-block:: json
+To see first page of paginated list of payments with two payments on each page use the below snippet:
```
Comment (TheMadeleine) [inline docs/api/payments.rst]: `To see the first page of the paginated list of payments `

File: docs/api/payments.rst [hunk 10]
```diff
@@ -1,37 +1,72 @@
...
++===============+================+======================================+
+| Authorization | header         | Token received during authentication |
++---------------+----------------+--------------------------------------+
+| id            | url attribute  | Id of the requested payment          |
++---------------+----------------+--------------------------------------+
+
+Example
+^^^^^^^
+
+To see the details of the payment method with ``id = 20`` use the below method:
```
Comment (tuka217) [inline docs/api/payments.rst]: of the payment (not method) with ...

File: docs/api/payments.rst [hunk 11]
```diff
@@ -1,37 +1,72 @@
...
-
-Response
-^^^^^^^^
+     $ curl http://demo.sylius.org/api/v1/payments/20 \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json"
+
+.. note::
+
+    The *id = 20* is just an example. Your value can be different.
```
Comment (tuka217) [inline docs/api/payments.rst]: Everywhere we have "The *20* id is just an example" or "The *potato* code is just an example"

**Comment by pjedrzejewski** [comment]
Thanks Łukasz!

**Pull request merged**